styles/tokens/promo.json

```diff
@@ -24,6 +33,33 @@
       "file": "settings",
       "themable": true
     },
+    "content-padding": {
+      "comment": "The padding for the content in the Promo component.",
+      "value": "{space.inset.l.value}",
+      "attributes": {
+        "category": "size"
+      },
+      "file": "settings",
+      "themable": true
+    },
+    "flag-content-padding": {
+      "comment": "The padding for the content in the Flag Promo component.",
+      "value": "{space.inset.m.value}",
+      "attributes": {
+        "category": "size"
+      },
+      "file": "settings",
+      "themable": true
+    },
```

Comment:
Minor thing but this can be grouped with the other `flag-` variables further down in the file.

Reply:
This one can't because the ones below are under image. This one is directly under promo and not for the image. Will group the others below though. Thank you!